azure-monitor-opentelemetry-distro/setup.cfg

```diff
@@ -1,45 +1,45 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License in the project root for
-# license information.
-# --------------------------------------------------------------------------
-[metadata]
-name = azure-monitor-opentelemetry-distro
-description = Azure Monitor Distro for OpenTelemetry Python
-long_description = file: README.md
-long_description_content_type = text/markdown
-author = Microsoft Corporation
-url = https://github.com/microsoft/ApplicationInsights-Python
-platforms = any
-license = MIT License
-classifiers =
-    Development Status :: 4 - Beta
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    License :: OSI Approved :: MIT License
-    Typing :: Typed
-
-[options]
-python_requires = >=3.6
-package_dir=
-    =src
-packages=find_namespace:
-zip_safe = False
-include_package_data = True
-install_requires =
-    azure-monitor-opentelemetry-exporter == 1.0.0b7
-    opentelemetry-instrumentation == 0.32b0
-
-[options.packages.find]
-where = src
-
-[options.entry_points]
-opentelemetry_distro =
-    azure_monitor_opentelemetry_distro = azure.monitor.opentelemetry.distro.distro:AzureMonitorDistro
-opentelemetry_configurator =
-    azure_monitor_opentelemetry_configurator = azure.monitor.opentelemetry.distro.configurator:AzureMonitorConfigurator
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+[metadata]
+name = azure-monitor-opentelemetry-distro
+description = Azure Monitor Distro for OpenTelemetry Python
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Microsoft Corporation
+url = https://github.com/microsoft/ApplicationInsights-Python
+platforms = any
+license = MIT License
+classifiers =
+    Development Status :: 4 - Beta
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    License :: OSI Approved :: MIT License
+    Typing :: Typed
+
+[options]
+python_requires = >=3.6
+package_dir=
+    =src
+packages=find_namespace:
+zip_safe = False
+include_package_data = True
+install_requires =
+    azure-monitor-opentelemetry-exporter == 1.0.0b7
```

Comment:
This line is the only difference. I am not sure why the differ is not catching this correctly.

Reply:
It's whitespace related -- use the gear and select hide whitespace worked for me.